index.html

```diff
@@ -1183,6 +1183,117 @@
+
+    These principles don't describe exactly how to distinguish acceptable
+    information from information we'd rather not expose. API designers instead
+    need to balance the harm to users from exposing information against the harm
+    to users from blocking that exposure. When in doubt, designers should ensure
+    that different user agents can help their users balance the costs in
+    different ways.
+
+The following subsections discuss how to review an API proposal that exposes data that
+provides a new way to infer each of the above categories of information. They explain how
+to <a data-cite="design-principles#leave-the-web-better">leave the web better than you
+found it</a>.
+
+### Handling acceptable information {#acceptable-information}
+
+<div class="practice">
+  <span class="practicelab">New APIs can add additional ways of getting acceptable
+  information that are guarded at least as strongly as the old ways.</span>
+</div>
+
+Acceptable information exposure is always qualified by the (possibly empty) set of
+user-controlled settings or permissions that guard access to it. For example, the URLs of
+resources, the timing of link clicks, and the referrer chain within a single origin are
+not guarded by anything; the scroll position is guarded by the setting to turn off
+javascript; and access to the camera or geolocation are guarded by permission prompts.
+
+Information that would be acceptable to expose under one set of guards might be
+unacceptable under another set, so when an API designer intends to explain that their new
+API is acceptable because an existing acceptable API already exposes the same information,
+they must be careful to ensure that their new API is only available under a set of guards
+that's at least as strict. Without those guards, they need to make the argument from
+scratch, without relying on the existing API.
+
+### Handling information that's being removed {#unacceptable-information}
+
+<div class="practice">
+  <span class="practicelab">If existing APIs provide access to some information, but
+  we have a plan to change those APIs to prevent that access, new APIs must not be added
+  that provide that same information without extra guards that make the information
+  acceptable.</span>
+</div>
+
+### Handling information we can't completely block {#sketchy-information}
+
+<div class="practice">
+  <span class="practicelab">New APIs that provide access to undesirable information should
+  not make that information easier to access.</span>
+</div>
+
+
+1. If future web platform changes make it possible to remove other access to the
+   undesirable information, it should be clear how to extend those changes to the proposed
+   API.
+
+1. If an existing browser does block access to the undesirable information, perhaps by
+   breaking some experiences on the Web that other browsers don't wish to break, it should
+   be clear how the more-private browser can also prevent the new API from exposing that
```

Comment:
WDYT of changing

> it should be clear how the more-private browser can also prevent the new API from exposing that information.

to 

> it should be clear how the more-private browser can also prevent the new API from exposing that information _without breaking additional sites or user experiences_.

I think some text tweak is needed here because its not usually difficult (unclear) to figure out how to prevent the new API from leaking the existing information; whats difficult is judging and minimizing the compat risks of blocking the new API

Reply:
SGTM, done.